### ISSUE TITLE
Lambda runtime nodejs18.x has expired, but the code for the check does not reflect this yet

### ISSUE THREAD

**Issue by RoryKiefer** [bridgecrewio/checkov#7300]
**Describe the issue**
[CKV_AWS_363](https://www.checkov.io/5.Policy%20Index/terraform.html#:~:text=511-,CKV_AWS_363,-resource) is not up to date and lacks already-[deprecated](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtimes-deprecated) lambda content

**Examples**
```
# Lambda function
resource "aws_lambda_function" "example" {

  runtime = "nodejs18.x" # <-- this was deprecated on 09/01/2025 but checkov doesn't report it as such
  # aws documentation: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtimes-deprecated

  filename         = data.archive_file.example.output_path
  function_name    = "example_lambda_function"
  role             = aws_iam_role.example.arn
  handler          = "index.handler"
  source_code_hash = data.archive_file.example.output_base64sha256

  environment {
    variables = {
      ENVIRONMENT = "production"
      LOG_LEVEL   = "info"
    }
  }

  tags = {
    Environment = "production"
    Application = "example"
  }
}
```

**Version (please complete the following information):**
 - Checkov Version: any

**Additional context**
 The solution is [here](https://github.com/bridgecrewio/checkov/pull/7284) and all thats needed is an un-comment and linter-auto-fix to implement. Note: This PR also addresses [issue # 7283](https://github.com/bridgecrewio/checkov/issues/7283).

**Comment by maxamel** [comment]
Right, can you update the PR with the needed versions? I see some versions are due.
